### PR TITLE
Choose screenshot format

### PR DESCRIPTION
**Pull request by ValentinLab**
I don't have macOS Big Sur to try it.
Tell me if everything is okay!

### CODE REVIEW

**Comment by ValentinLab** [comment]
I am going to fix it!

File: defaults.yml [hunk 1]
```diff
@@ -243,6 +243,19 @@ categories:
           - value: false
             text: Do not display the thumbnail
         versions: [Big Sur, Catalina]
+sc      - key: set-image-format
```
Comment (yannbertrand) [inline defaults.yml:246]:
```suggestion
      - key: set-image-format
```

File: defaults.yml [hunk 2]
```diff
@@ -243,6 +243,19 @@ categories:
           - value: false
             text: Do not display the thumbnail
         versions: [Big Sur, Catalina]
+      - key: set-image-format
+        domain: com.apple.screencapture
+        title: Choose screenshot format
+        description: Choose the image format of the screenshots.
```
Comment (yannbertrand) [inline defaults.yml:249]:
```suggestion
        description: Choose the screenshots image format.
```

File: defaults.yml [hunk 3]
```diff
@@ -243,6 +243,19 @@ categories:
...
+        param:
+          type: string
+        examples:
+          - value: png
+            default: true
+            text: The image generated is a png
```
Comment (yannbertrand) [inline defaults.yml:255]:
```suggestion
            text: The generated image is a png
```

File: defaults.yml [hunk 4]
```diff
@@ -243,6 +243,19 @@ categories:
...
+          - value: jpg
+            text: The image generated is a jpg
```
Comment (yannbertrand) [inline defaults.yml:257]:
```suggestion
            text: The generated image is a jpg
```

File: defaults-fr.yml [hunk 5]
```diff
@@ -243,6 +243,19 @@ categories:
...
+        description: Choisir le format de fichier de l'image dans lequel la capture est enregistrée.
+        param:
+          type: string
+        examples:
+          - value: png
+            default: true
+            text: L'image générée est au format png
+          - value: jpg
+            text: L'image générée est au format jpg
+        versions: [Catalina]
```
Comment (yannbertrand) [inline defaults-fr.yml:258]:
```suggestion
        versions: [Big Sur, Catalina]
```

File: defaults.yml [hunk 6]
```diff
@@ -243,6 +243,19 @@ categories:
...
+        description: Choose the screenshots image format.
+        param:
+          type: string
+        examples:
+          - value: png
+            default: true
+            text: The generated image is a png
+          - value: jpg
+            text: The generated image is a jpg
+        versions: [Catalina]
```
Comment (yannbertrand) [inline defaults.yml:258]:
```suggestion
        versions: [Big Sur, Catalina]
```

File: defaults-fr.yml [hunk 7]
```diff
@@ -243,6 +243,19 @@ categories:
           - value: false
             text: Ne pas afficher la vignette
         versions: [Big Sur, Catalina]
+      - key: set-image-format
```
Comment (yannbertrand) [inline defaults-fr.yml:246]:
```suggestion
      - key: type
```

File: defaults.yml [hunk 8]
```diff
@@ -243,6 +243,19 @@ categories:
           - value: false
             text: Do not display the thumbnail
         versions: [Big Sur, Catalina]
+      - key: set-image-format
```
Comment (yannbertrand) [inline defaults.yml:246]:
```suggestion
      - key: type
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]
Thank you for the contribution! 😄 

The `key` attribute means the defaults command "key": `defaults write <domain> <key> -<paramType> <param>`.

Can you confirm the command is `defaults write com.apple.screencapture type -string "png"`?

**Comment by ValentinLab** [comment]
Sorry, yes, you're right!

**Review by yannbertrand — APPROVED** [review]

**Comment by yannbertrand** [comment]
Fix #34

Thanks Valentin!

**Pull request merged**